src/NServiceBus.Transport.Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -139,5 +139,86 @@ internal static bool GetShouldExecuteInstaller(this SettingsHolder settings)
             }
             return true;
         }
+
+        /// <summary>
+        /// Disables the storing of undeliverable messages in the dead letter queue
+        /// </summary>
+        /// <param name="config"></param>
+        public static void DoNotUseDeadLetterQueue(this TransportExtensions<MsmqTransport> config)
+        {
+            Guard.AgainstNull(nameof(config), config);
+            config.GetSettings().Set("UseDeadLetterQueue", false);
+        }
+
+        internal static bool GetUseDeadLetterQueue(this SettingsHolder settings)
+        {
+            return settings.GetOrDefault<bool>("UseDeadLetterQueue");
+        }
+
+        /// <summary>
+        /// Does not cache connections. 
```

Comment:
This comment feels a bit like "Here we enter a loop" just above a "for" loop ;-)
Maybe add something that implies what the difference is?

Reply:
Added now